app/stac_api/views.py

```diff
@@ -44,6 +44,13 @@ def get_collection_etag(request, *args, **kwargs):
     The ETag is an UUID4 computed on each object changes (including relations; provider and links)
     '''
     tag = get_etag(Collection.objects.filter(name=kwargs['collection_name']))
+    # uncomment the following lines to see the SQL query and the EXPLAIN.. ANALYZE
+    # output:
+    # logger.debug(
+    #     Collection.objects.filter(name=kwargs['collection_name']
+    #                              ).explain(verbose=True, analyze=True)
+    # )
+    # logger.debug(Collection.objects.filter(name=kwargs['collection_name']).query)
```

Comment:
I just had idea, instead of commenting these lines out, we could simply add a settings to enable them, e.g. 
```python
if settings.DEBUG_ENABLE_DB_EXPLAIN_ANALYZE:
   logger.debug ....
```
The settings would be `False` by default and could be set by environment variable. This way we could always enable this logs even in already deployed version if needed. @boecklic what do you think ?

Reply:
Implemented

Reply:
We can have it as additional option, yes. What I find more intuitive in general (other than many different flags to turn certain debug msg on and off) is just having more than one (e.g. 2 or 3) debug levels. DEBUG1 is 'just enough to retrace what's happening' on prod, DEBUG2 is 'some more stuff helpful for development', DEBUG3 is 'really everything'... these messages could then go to level 3

Reply:
That would make sense @boecklic. I'd suggest to do this kind of "debugging triage" in a separate PR, as probably debug prints other than only from this PR might be affected. Probably a dedicated ticket for this cleaning up and triage might make sense. So for the moment, we could merge this PR as is and re-think our debugging levels in general in a next step.
What do you think?

Reply:
Sure, no hurry on that, if agreed we can preview this for post ogd cleanup